huaweicloud/services/as/resource_huaweicloud_as_configuration.go

```diff
@@ -279,9 +265,6 @@ func buildDiskOpts(diskMeta []interface{}) ([]configurations.DiskOpts, error) {
 		size := disk["size"].(int)
 		volumeType := disk["volume_type"].(string)
 		diskType := disk["disk_type"].(string)
-		if err := validateDiskSize(size, diskType); err != nil {
```

Comment:
If you remove the validation, the second parameter of the response has no meaning, please discard it.

Reply:
fixed.